docs/features/manual-dependencies.md

```diff
@@ -153,3 +153,23 @@ remote-dependencies:
     description: Provides foo and a helpful interface around foo-like tasks.
     homepage: https://www.foowrapper-home.com
 ```
+
+## Performance
+
+The FOSSA service caches the results of dependency analysis by different keys depending on the type of remote dependency specified (explained below).
```

Comment:
[nit] "Different keys" sounds a little awkward, I'm not sure the idea of a "key" is immediately obvious. I would maybe remove this part, or say something more generic like "we cache things in different ways depending on the type".

Reply:
Good call! Updated!